Explains how to chill validators and nominators and what it means (#756)
w3f/polkadot-wiki: docs/maintain-guides-how-to-chill.md

---
id: maintain-guides-how-to-chill
title: Chilling

> sidebar_label: Chilling

Reviewer: Should we rename this to something a bit more specific?

Reviewer: Such as "How to chill on Network"?

Reviewer 2: I would suggest "How to Chill :sunglasses: "

(or How to Chill when Staking)

---

> Stakers can assume any one of the three states: validating, nominating, or chilling. When a staker

Reviewer: I think this should also be included on the [Staking](https://wiki.polkadot.network/docs/en/learn-staking) page to avoid confusion.

Reviewer: perhaps in the [chilling section](https://wiki.polkadot.network/docs/en/learn-staking#chilling) of the staking page, we could link this new page for more reading :)

Reviewer 2: I think having it's own page would be nice. Then link it from the staking one.

does not want to actively engage in staking - but does not want to unbond their funds either - they
can choose to temporarily become inactive while keeping their funds staked. The act of stopping
either nominating or validating is known as _chilling_.

An account can step back from participating in active staking by calling the `chill` extrinsic in
the [staking pallet][chill extrinsic]. When an account chooses to chill, they will become inactive

> in the next era. The call must be signed by the _controller_ account, not the _stash_.

Reviewer: Do you want to include a photo here too? That way users can reference this?

> in the next era. The call must be signed by the _controller_ account, not the _stash_.

Reviewer: Why must the call be from the controller account and not either the controller or stash account?

Reviewer 2: The controller account is the only account that can do staking actions (bond, unbond, stake, chill, etc). The stash account I suppose can submit this as well, but only if it is bonded to itself (and acts as a controller as well).

Reviewer: Ah! That makes sense, I totally forgot about the roles there. Maybe @lsaether, do you want to add in a reminder or possibly a link to the accounts page about the controller and stash roles?

Author: Great idea.

## Chilling as a Nominator

When you chill after being a nominator, your nominations will be reset. This means that when you

> decide to start nominating again you will need to select validators to nominate once again. These

Reviewer: To go from chilling to being active again, is there any steps that should be taken? Or can I just start nominating or validating again?

Reviewer 2: Yep, just go through the process of nominating again (you don't need to bond or set controller in this case).

Reviewer: It might help @lsaether to include these details in this section here, but what do you think?

Reviewer 2: Yeah, might help to explicitly mention it.

can be the same validators if you prefer, or a completely new set. Just be aware - your nominations
will not persist across chills.

## Chilling as a Validator

> When you chill after being a validator, your nominators will not automatically go away. As long as

Reviewer 2: ```suggestion
When you voluntarily chill after being a validator, your nominators will not automatically go away. As long as
```

> your nominators make no action, you will still have the nominations when you choose to become an

Reviewer: I'm just wondering what happens if you're a chilling validator and your nominators do make action?

Reviewer 2: If they nominate somebody else, those nominations will take priority when the validator comes back.

Reviewer: This might be helpful to add into this paragraph as another sentence.

> active validator once again. When you become an active validator you will also need to re-set your

Reviewer: ```suggestion
active validator once again. When you become an active validator you will also need to reset your
```

> validator preferences (comission, etc.). These can be configured as the same values that were set

Reviewer: ```suggestion
validator preferences (commission, etc.). These can be configured as the same values that were set
```
